helm_chart/ibm_storage_enabler_for_containers/values.yaml

```diff
@@ -0,0 +1,75 @@
+# ----------------------------------------------------
+# Helm chart to install IBM storage Enabler for Containers.
+# ----------------------------------------------------
+
+images:
+  ubiquity: ibmcom/ibm-storage-enabler-for-containers:2.0.0
+  ubiquitydb: ibmcom/ibm-storage-enabler-for-containers-db:2.0.0
+  provisioner: ibmcom/ibm-storage-dynamic-provisioner-for-kubernetes:2.0.0
+  flex: ibmcom/ibm-storage-flex-volume-for-kubernetes:2.0.0
+
+spectrumConnect:
+  connectionInfo:
+    ## IP\FQDN and port of Spectrum Connect server.
+    fqdn:
+    port:
+
+    ## Username and password defined for IBM Storage Enabler for Containers interface in Spectrum Connect.
+    username:
+    password:
+
+    # SSL verification mode. Allowed values: require (no validation is required) and verify-full (user-provided certificates).
+    sslMode: require
+
+  backendConfig:
+    # A prefix for any new volume created on the storage system.
+    instanceName:
```

Comment:
instanceName is confusing, it is a prefix, what about something like "volumePrefix"?

Reply:
I agree that instanceName is not self explanatory, and volumePrefix could be an alternative.
For now lets keep it instanceName, and raise this aspect in the design so we will talk about it.
I am not sure i want to change it to volumePrefix, because it may confuse inside the templates (which hard to change in order to keep backword compatibility).
But let talk about it offline.